src/olympia/abuse/models.py

```diff
@@ -304,33 +305,43 @@ def process_decision(
         abuse_report_or_decision = (
             self.appealed_decisions.first() or self.abusereport_set.first()
         )
-        decision = ContentDecision.objects.create(
```

Comment:
Curious, why not just add a return if the decision exists?

```python
if ContentDecision.objects.filter(cinder_id=decision_cinder_id).exists:
  return
```

Maybe there is a slight race condition there where you could still get the error, so you could just add a try catch for integritry error? 

```python
try:
    decision = ContentDecision.objects.create(
        ...
    )
except IntegrityError.Duplicate:
    return
```

I know its basically equivalent it just seems like using the get or create adds an uglier call.. I almost changed my mind while writing this but curious to hear your thoughts on it?